src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -93,7 +93,7 @@ public ShooterSubsystem() {
     shooterMAXRight.setIdleMode(IdleMode.kCoast);
     shooterMAXLeft.setSmartCurrentLimit(currentLimit);
     shooterMAXRight.setSmartCurrentLimit(currentLimit);
-    // Sets the left shooter motor to follow the right motor, and be inverted.
```

Comment:
Try not to mix unrelated cleanup.

Reply:
Got it.  For future reference, don't fix unrelated issues you spot while working on a feature branch that are not part of said branch.  Merge conflicts are hell.